tracer/src/Datadog.Trace/ClrProfiler/AutoInstrumentation/AWS/StepFunctions/IAmazonStepFunctionsRequestWithStateMachineArn.cs

```diff
@@ -0,0 +1,20 @@
+// <copyright file="IAmazonStepFunctionsRequestWithStateMachineArn.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+#nullable enable
+
+namespace Datadog.Trace.ClrProfiler.AutoInstrumentation.AWS.StepFunctions
+{
+    /// <summary>
+    /// Interface for ducktyping IAmazonStepFunctionsRequest implementations
+    /// </summary>
+    internal interface IAmazonStepFunctionsRequestWithStateMachineArn
```

Comment:
A bit nit-picky, but can we use `Aws` instead of `Amazon`? They are branded as [AWS Step Functions](https://aws.amazon.com/step-functions/).